src/main/java/com/example/demo/controller/MovieController.java

```diff
@@ -0,0 +1,169 @@
+package com.example.demo.controller;
+
+import com.example.demo.entity.Booked;
+import com.example.demo.entity.BookedSeat;
+import com.example.demo.entity.Movie;
+import com.example.demo.repository.IMovieRepository;
+import com.example.demo.service.MovieService;
+import lombok.AllArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+import java.util.regex.Pattern;
+
+
+@RestController("movies")
+@AllArgsConstructor
+public class MovieController {
+
+    final IMovieRepository movieRepository;
+
+    final MovieService movieService;
+
+
+    public static Boolean nameVerification(String name) {
+        String regex = "[A-ZĄĆĘŁŃÓŻŹ][a-ząćęłńóżź][a-ząćęłńóżź]+ " +
+                "[A-ZĄĆĘŁŃÓŻŹ][a-ząćęłńóżź][a-ząćęłńóżź]+(-[A-ZĄĆĘŁŃÓŻŹ][a-ząćęłńóżź][a-ząćęłńóżź]+)?";
+        Pattern pattern = Pattern.compile(regex);
```

Comment:
kompilujemy regex za każdym razem jak jest potrzebny - czy to wydajne?

Reply:
Nie bardzo. Lepszym rozwiązaniem byłoby pewnie zdefiniowanie tego Stringa regex, jako np. final static w klasie movieController, poza metodą.